src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -226,49 +221,26 @@ export const SPECIAL_FIELDS = {
       );
     },
   },
-  csp: {
-    fields: ['issue_title', 'project.name', 'latest_event'],
-    sortField: 'issue_title',
-    renderFunc: (data, {organization, location}) => {
-      const target = {
-        pathname: `/organizations/${organization.slug}/events/`,
-        query: {
-          ...location.query,
-          eventSlug: `${data['project.name']}:${data.latest_event}`,
-        },
-      };
-      return (
-        <Container>
-          <Link css={overflowEllipsis} to={target} aria-label={data.issue_title}>
-            {data.issue_title}
-          </Link>
-        </Container>
-      );
-    },
-  },
-  event_count: {
-    title: 'events',
-    fields: ['event_count'],
-    sortField: 'event_count',
+  // TODO generalize this.
+  'count(id)': {
+    sortField: 'count_id',
```

Comment:
Can you only sort by one column?

Reply:
The links only add/change a single sort condition.

Reply:
Ok. For events we use `-timestamp, -event_id` or `timestamp, event_id` so we have a stable order, which we don't have here. Not sure if it matters though.